exe-unit/src/bin.rs

```diff
@@ -315,6 +315,10 @@ async fn run() -> anyhow::Result<()> {
 
 #[actix_rt::main]
 async fn main() {
+    std::panic::set_hook(Box::new(|e| {
+        log::error!("ExeUnit Supervisor panic: {e}");
+    }));
```

Comment:
Does this mean the binary won't print backtrace even with `RUST_BACKTRACE` set? This would limit the usefulness of logs supplied by users.

Nevertheless, this change deserves a comment with some explaination.

Reply:
On the contrary - panics were not present in logs at all, just printed to stderr, which we would miss while retrieving logs from users.

I've modified the code to also execute the default panic hook

Reply:
ack